src/mon/MonCommands.h

```diff
@@ -155,6 +155,7 @@ COMMAND("pg ls-by-osd " \
 	"list pg on osd [osd]", "pg", "r", "cli,rest")
 COMMAND("pg ls " \
         "name=pool,type=CephInt,req=false " \
+        "name=osd,type=CephInt,req=false " \
```

Comment:
i don't think "pg ls" command accepts three optional arguments. it does accept two optional argument.

the reason why `else if (prefix == "pg ls") {` branch checks for the "pool" is just that it is reused by `"pg ls-by-pool"`, and `"pg ls-by-pool"` **sets** the "pool" argument programmatically:
```c++
    cmd_putval(g_ceph_context, cmdmap, "pool", pool);
```